sale_order_line_price_history/views/sale_views.xml

```diff
@@ -9,15 +9,18 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='order_line']/tree">
                 <field name="order_partner_id" invisible="1" />
-                <widget
-                    name="sale_line_price_history_widget"
-                    width="20px"
+                <field
+                    name="id"
+                    string="Price Widget"
+                    context="{'active_id': id, 'active_ids': [id]}"
                     attrs="{'invisible': [('id', '!=', False)]}"
+                    widget="sale_line_price_history_widget"
                 />
                 <button
                     name="%(sale_order_line_price_history_action)d"
                     type="action"
-                    icon="fa-clock-o"
+                    icon="fa-history"
```

Comment:
Why changing the icon?

Reply:
Because this icon represent better the main purpose of this button, show historical prices

Reply:
OK, but the button should go out and it should be inside the widget.